db.json

```diff
@@ -0,0 +1,215 @@
+{
+  "workstreams": [
+    {
+      "id": "compat",
+      "name": "Compatibility",
+      "scope": "CSS properties and DOM APIs that web browsers need to support for compatibility with the de facto web.",
+      "editors": [
+        { "name": "Mike Taylor", "email": "miket@mozilla.com" }
+      ],
+      "standards": [
+        {
+          "title": "Compatibility",
+          "ref-title": "COMPAT",
+          "href": "https://compat.spec.whatwg.org/",
+          "description": "The Compatibility Standard describes a collection of non-standard (and often vendor-prefixed) CSS properties and DOM APIs that web browsers need to support for compatibility with the de facto web.",
+          "authors": []
```

Comment:
Is the idea that authors inherits from workstream editors automatically, or did you just not fill this out yet? The cases like HTML are the tricky ones.

Reply:
Yeah, I was thinking authors would add to editors.

Reply:
In theory we could have an "editor" responsible for logistics/final decision calls that is not an "author". Maybe not worth worrying about, though.

Reply:
If we do make them additive then I think reflecting that in the field name would be good, e.g. `additional_authors` or `non_editor_authors`.